packages/node/src/login/oidc/oidcHandlers/RefreshTokenOidcHandler.ts

```diff
@@ -183,6 +187,7 @@ export default class RefreshTokenOidcHandler implements IOidcHandler {
       oidcLoginOptions.dpop,
       oidcLoginOptions.keepAlive ?? true,
       keyPair,
+      oidcLoginOptions.eventEmitter,
```

Comment:
This missing was a bug resulting in the session expiration event not being fired.